### DISCUSSION TITLE
[Projects Beta] Cannot specify "draft:false" in the Projects workflow query

### DISCUSSION THREAD

**Discussion by RudyLu** [community/community#49279] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

When trying to create a new workflow in a project like this,

<img width="1153" alt="Screenshot 2023-03-06 at 3 27 01 PM" src="https://user-images.githubusercontent.com/1704629/223045239-9238fb2f-8982-49f3-abdd-7a2a3a7372aa.png">

it would pop up an error message after pressing the save button,
<img width="428" alt="Screenshot 2023-03-06 at 3 27 07 PM" src="https://user-images.githubusercontent.com/1704629/223045356-d259cba3-c507-4729-87d5-212666795fe3.png">

**Comment by MatShir** [comment]
Same here !

**Comment by octatone** [comment, marked as answer]
👋 Hi there, you should be able to achieve the desired filter by using the `is` qualifier: `is:pr -is:draft`
